tests/test_unit/test_console.py

```diff
@@ -640,3 +640,71 @@ def test_standardize_in_place_multi_noop(self) -> None:
                 self.assertEqual(contents, exp)
         finally:
             any(map(os.unlink, tmpfnames))
+
+    def test_standardize_target_encoding(self) -> None:
+        table: TableType = [["Å", "B", "C"], ['é', 'ü', '中'], [4, 5, 6]]
+        dialect = SimpleDialect(delimiter=";", quotechar="", escapechar="")
+        encoding='utf-8'
+        tmpfname = self._build_file(table, dialect, encoding=encoding)
+
+        tmpfd, tmpoutname = tempfile.mkstemp(prefix="ccsv_", suffix=".csv")
+        os.close(tmpfd)
+
+        application = build_application()
+        tester = Tester(application)
+        tester.test_command("standardize", ["-o", tmpoutname, '-E', 'utf-8', tmpfname])
+
+        # Excel format (i.e. RFC4180) *requires* CRLF
+        crlf = "\r\n"
+        exp = crlf.join(["Å,B,C", "é,ü,中", "4,5,6", ""])
+        with open(tmpoutname, "r", newline="") as fp:
+            output = fp.read()
+
+        try:
+            self.assertEqual(exp, output)
+        finally:
+            os.unlink(tmpfname)
+            os.unlink(tmpoutname)
+
+    def test_standardize_target_encoding2(self) -> None:
+        table: TableType = [["A", "B", "C"], ['é', 'è', 'à'], [4, 5, 6]]
+        dialect = SimpleDialect(delimiter=";", quotechar="", escapechar="")
+        encoding='latin-1'
+        tmpfname = self._build_file(table, dialect, encoding=encoding)
```

Comment:
These are really nice tests, thank you for adding them! I just have a question about this one (not a comment, feel free to ignore). Do you think it's worth adding a check that this works as intended, like this:
```python
from clevercsv.encoding import get_encoding

self.assertEqual("ISO-8859-1", get_encoding(tmpfname, try_cchardet=False))
```
and on line 689 below:
```python
self.assertEqual("utf-8", get_encoding(tmpoutname, try_cchardet=False))
```
Or would that be overkill?

Reply:
Yes, very good idea :+1: , I have added those assertions. [ac55544](https://github.com/alan-turing-institute/CleverCSV/pull/118/commits/ac5554408b56cc7fe7016d92eaef59521604f379)